cw/tvla_cfg_otbn_batch..yaml

```diff
@@ -0,0 +1,18 @@
+project_file: projects/opentitan_otbn_vertical_keygen.cwp
+trace_file: null
+trace_start: null
+trace_end: null
+leakage_file: null
+save_to_disk: null
+round_select: null
+byte_select: null
+input_file: null
+output_file: null
+number_of_steps: 1
+ttest_step_file: null
+plot_figures: true
+general_test: true
+mode: otbn
+key_len_bytes: 40
+sample_start: null
+num_samples : 3500
```

Comment:
Do we need a separate configuration file for the batch mode?
It seems that we can continue using `tvla_cfg_otbn.yaml`